targets/rapidjson/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM ossfuzz/base-libfuzzer
+MAINTAINER aarongreen@google.com
+RUN git clone https://github.com/miloyip/rapidjson.git
+RUN git clone https://github.com/json-schema-org/JSON-Schema-Test-Suite.git
+COPY build.sh /src/
+COPY parse_fuzzer.cc /src/rapidjson
```

Comment:
since parse_fuzzer comes not from rapidjson, it is best to put it right into /src/.